README.md

```diff
@@ -192,6 +192,15 @@ The typical flow about how to use this gem is:
 
 
 ## Notes on upgrading
+
+##### Upgrading from 2.2.0 to 3.2.0
+
+If you are upgrading you need to rename your `rapidfire_question_groups` to `rapidfire_surveys` and `rapidfire_answer_groups` to `rapidfire_attempts`. Run the given task to do that for you.
+
+```shell
+    $ rake rapidfire:upgrade:migrations:from210to330
```

Comment:
rake task says from `210 to `330`, and heading says from `2.2.0 to `3.2.0`. the release that we are going to make is `3.0.0` :D